src/metadata/metadata.c

```diff
@@ -1225,7 +1225,13 @@ static void _recovery_rebuild_metadata(ocf_pipeline_t pipeline,
 
 	for (cline = 0; cline < collision_table_entries; cline++) {
 		ocf_metadata_get_core_info(cache, cline, &core_id, &core_line);
+		if (!ocf_metadata_check(cache, cline) ||
+				core_id > OCF_CORE_MAX) {
```

Comment:
should `OCF_CORE_MAX` be included as a valid value? This value is also `OCF_CORE_ID_INVALID`. Maybe you meant `OCF_CORE_ID_MAX`?

Reply:
OCF_CORE_MAX is a special value indicating unused mapping and should not result in cache recovery error